ModUtils/RoomUtils.cs

```diff
@@ -9,5 +14,296 @@ public static IEnumerable<UndertaleRoom> GetRooms()
         {
             return ModLoader.Data.Rooms;
         }
+        public static UndertaleRoom GetRoom(string name)
+        {
+            try
+            {
+                UndertaleRoom room = ModLoader.Data.Rooms.First(t => t.Name.Content == name);
+                Log.Information($"Found room {name}");
+                return room;
+            }
+            catch
+            {
+                throw;
+            }
+        }
+        public static UndertaleRoom AddRoom(string name)
+        {
+            UndertaleRoom room = new()
+            {
+                Name = ModLoader.Data.Strings.MakeString(name),
+                Caption = ModLoader.Data.Strings.MakeString(""),
+                Flags = UndertaleRoom.RoomEntryFlags.IsGMS2 ^ UndertaleRoom.RoomEntryFlags.EnableViews,
+            };
+            ModLoader.Data.Rooms.Add(room);
```

Comment:
What happens if trying to add a room which already exists ?

Reply:
Error handling added